Gaming The Syllabus/Assets/Scripts/Combat/UI/Healthbar.cs

```diff
@@ -23,6 +34,27 @@ public void UpdateUnitName(string newText)
         unitName.text = newText;
     }
 
+
+    /// <summary>
+    /// Updates the text seen below a Combat Unit's health bar. For example, if a combat unit
+    /// takes 5 damage then a 5 will popup on screen below their health.
+    /// </summary>
+    /// <param name="amtChanged">Make sure you pass in a string represented integer. For example, "5" or "-5"</param>
+    public void UpdateHealthChangeText(string amtChanged)
+    {
+        if (int.Parse(amtChanged) < 0) {
+            healthChangeText.color = Color.red;
+            healthChangeText.text = amtChanged;
+        } else if (int.Parse(amtChanged) > 0) {
+            healthChangeText.text = "+" + amtChanged;
+            healthChangeText.color = Color.green;
+        }
+    }
+
+    public void ShowHealthChangeText() {
```

Comment:
Is there ever a time we'd wanna show this without the unit being damaged? If not might make sense to just make this private and call it when UpdateHealthChangeText is called